root/main/ConfirmSeed.js

```diff
@@ -29,6 +29,17 @@ const ConfirmSeed = ({
 }: Props): React$Element<typeof Layout> => {
   const $c = React.useContext(SanitizedCatalystContext);
   const title = l('Confirm Form Submission');
+
```

Comment:
I spent a lot of time trying to submit the form directly from this component rather than clicking the button in `ConfirmSeedButtons`, but `React.useEffect` calls that I add here strangely never seem to execute. This is my first time using React, so presumably there's some subtlety that I'm missing.

Reply:
That would be because `ConfirmSeedButtons` is "hydrated" on the client (effects and event handlers are run in the browser), while `ConfirmSeed` is only run server-side.

Reply:
Thanks, that was my guess. :-)